### ISSUE TITLE
Order of facets

### ISSUE THREAD

**Issue by patrickzurek** [eXtensibleCatalog/Drupal-Toolkit#251]
JIRA issue created by: dlindahl
Originally opened: 2011-09-26 04:20 PM

Issue body:
 (nt)

**Comment by patrickzurek** [comment]
JIRA Comment by user: dlindahl
JIRA Timestamp: 2011-09-26 04:20 PM

Comment body:

I am doing some screen captures for my demo and I did not like the order of the facets.  I logged into the admin screen and reordered them, and changed which ones were collapsed.  Then, I verified that my settings worked and they did (as long as I was still logged in as the administrator).

But, when I logged out, the facets order and collapsed/not collapsed reverted back to the old settings (they appear in the old order).

I cleared my browser cache and it did not help.

When I login as admin, I see the facets in the correct order.
When I logout, I see them in the old order.
